prov/gni/contrib/gnitest.supp

```diff
@@ -0,0 +1,46 @@
+{
+   ioctl_cq_create
+   Memcheck:Param
+   ioctl(generic)
+   fun:ioctl
+   fun:cq_create
+   ...
+}
+{
+   ioctl_GNI_MemRegister
+   Memcheck:Param
+   ioctl(generic)
+   fun:ioctl
+   fun:GNI_MemRegister
+   ...
+}
+{
+   ioctl_GNI_EpPostDataWId
+   Memcheck:Param
+   ioctl(generic)
+   fun:ioctl
+   fun:GNI_EpPostDataWId
+   ...
+}
+{
+   ioctl_gni_fma_assign
+   Memcheck:Param
+   ioctl(generic)
+   fun:ioctl
+   fun:gni_fma_assign
+   ...
+}
+{
+   GNI_EpPostDataTestById__gnix_dgram_poll
+   Memcheck:Addr4
+   fun:GNI_EpPostDataTestById
+   fun:_gnix_dgram_poll
+   ...
+}
+{
+   GNI_PostDataProbeById__gnix_dgram_poll
+   Memcheck:Addr4
+   fun:GNI_PostDataProbeById
+   fun:_gnix_dgram_poll
+   ...
+}
```

Comment:
This is a valgrind suppression file for running valgrind on the unit tests (and probably most applications that will use the gni provider).  A contrib directory is the typical location for such a file.